spec/controllers/charts_controller_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Account::ChartsController, type: :controller do
+  describe "GET requests to actions" do
+    it "returns a 200 status code" do
+      get :users_by_week
+      expect(response).to have_http_status(302)
```

Comment:
test for 302? this is a joke? what are you testing here?

Reply:
`be_redirect` or `redirect_to`

Reply:
check for assigns at least